sim/src/pmodel/mod.rs

```diff
@@ -0,0 +1,55 @@
+mod prob;
+
+pub use prob::{proba_decaying_sigmoid, erf_distrib_bounded};
+use geom::{Duration};
+
+pub enum SEIR {
+    Sane,
+    Exposed,
+    Infectious,
+    Recovered,
+}
+
+impl SEIR {
+    const T_INF: f64 = 3600.0 * 10.0; // TODO dummy values
+    const T_INC: f64 = 3600.0; // TODO dummy values
+    const R_0: f64 = 2.5;
+    const S_RATIO: f64 = 0.985;
+    const E_RATIO: f64 = 0.005;
+    const I_RATIO: f64 = 0.01;
+    const R_RATIO: f64 = 0.0;
+
+    // TODO change that name it's bad
+    pub fn get_transition_time_from(state: Self) -> Duration {
+        match state {
+            SEIR::Sane => Duration::seconds(SEIR::T_INF / SEIR::R_0),
+            SEIR::Exposed => Duration::seconds(SEIR::T_INC),
+            SEIR::Infectious => Duration::seconds(SEIR::T_INF),
+            SEIR::Recovered => {
```

Comment:
```suggestion
            SEIR::Recovered => unreachable!()
```
Optional suggestion. The error message is less informative, but it'll still point to this line number, so if it does happen, finding out the cause is easy.

Reply:
makes sense.